CryptoPrices/Sources/MyCoin/Sources/MyCoin/MyCoinView/MyCoinView.swift

```diff
@@ -8,28 +8,39 @@ import DomainTestHelpers
 import Styleguide
 import SwiftUI
 
+typealias Section = Styleguide.Section
```

Comment:
This looks like a repeated `typealias` declaration, I think we can define one in the `Styleguide` module directly and just use it here and for `HomeView` also. 🙏

Reply:
The `Section` belongs to the `Styleguide`, so if we want to define a `typealias` in the `Styleguide`, it must have a different name like `ListSection`, `CustomSection`, or something like that. But it doesn't make sense, we can just name it `struct ListSection` or `struct CustomSection`, so no `typealias` is necessary in the first place 🙏 

To be honest, those names are quite redundant and too little too specific. For e.g. let take the `ListSection` name as an example. `Section` can be used in any container view like `VStack`, `HStack` or even nested in another `Section`, not only in the `List`. So the name `ListSection` is not only too specific but also not totally correct 🙏 

I'd prefer to keep the original name as `Section`. And it's up to the modules which use the `Section` (`Home` and `MyCoin`) to define whatever `typealias` they want to be to avoid ambiguity with `SwiftUI`'s `Section`. For e.g in the `Home` it could be `typealias HomeSection = Styleguide.Section`, while in the `MyCoin` it could be `typealias MyCoinSection = Styleguide.Section`.

So instead of seeing it as the repetitive, another way to think about it is that, it just happens (coincidentally 😅) that 2 module `Home` and `MyCoin` define their own `typealias` with the same name `typealias Section = Styleguide.Section`. What do you think 🙏 ?

Reply:
Ohh you are right that we don't need a typealias anymore if it is placed in the `Styleguide` package, my bad 🙇‍♂️ Yep for sure there is not a need for renaming them to `HomeSection` or `MyCoinSection`, keeping just `Section` is fine as it is referring to the same struct in our common views 👍

Now, since the typealias is only used one time e.g. in [MyCoinView.swift](https://github.com/nimblehq/nimble-crypto-ios/pull/93/files/8f75a4712ac5c779545a99e5422c1a75c970746a#diff-ac6dd79a4e1836b63d9a2b484a757a66804ab87e1271482f22d347a2cf06d1d2) for `MyCoin` package, what do you think about turning it to private instead? Later on, when there is a new need for using it in multiple places, it would be better to move it out of the `MyCoinView` class and place it in a new file for shared usage 😁

Reply:
@minhnimble Yes, totally agree 👍
Will make both `typealias`es in the `Home` and `MyCoin` private!

Reply:
Updated in https://github.com/nimblehq/nimble-crypto-ios/pull/93/commits/13771ebb808b2a92d7319399586989f1f2a8472b 🙏